test/geometry/GeoParams.test.cc

```diff
@@ -0,0 +1,39 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file GeoParams.test.cc
+//---------------------------------------------------------------------------//
+#include "geometry/GeoParamsTest.hh"
+
+// using namespace celeritas;
+using namespace celeritas_test;
+
+//---------------------------------------------------------------------------//
+// HOST TESTS
+//---------------------------------------------------------------------------//
+
+class GeoParamsHostTest : public GeoParamsTest
+{
+  public:
+    void SetUp() override
+    {
+        host_view = params()->host_view();
```

Comment:
For member functions (and especially base-class functions) it improves readability to access using `this->`. Could you change to `this->params()` here and in the other places it's used?

Reply:
Done.